README.md

```diff
@@ -106,34 +73,187 @@ To install [@okta/okta-auth-js](https://www.npmjs.com/package/@okta/okta-auth-js
 npm install @okta/okta-auth-js --save
 ```
 
-After running `npm install`:
-
-The minified auth client will be installed to `node_modules/@okta/okta-auth-js/dist`. You can copy the `dist` contents to a publicly hosted directory. However, if you're using a bundler like [Webpack](https://webpack.github.io/) or [Browserify](http://browserify.org/), you can simply import the module using CommonJS.
+After running `npm install`, the minified auth client will be installed to `node_modules/@okta/okta-auth-js/dist`. You can copy the `dist` contents to a publicly hosted directory. However, if you're using a bundler like [Webpack](https://webpack.github.io/) or [Browserify](http://browserify.org/), you can simply import the module using CommonJS.
 
 ```javascript
 var OktaAuth = require('@okta/okta-auth-js');
 var authClient = new OktaAuth(/* configOptions */);
 ```
 
-# API
+If you're using a bundler like webpack or browserify, we have implementations for jquery and reqwest included. To use them, import the SDK like this:
+
+```javascript
+// reqwest
+var OktaAuth = require('@okta/okta-auth-js/reqwest');
 
-## new OktaAuth(config)
+// jquery
+var OktaAuth = require('@okta/okta-auth-js/jquery');
+```
 
-Creates a new instance of the Okta Auth Client with the provided options. The client has many [config options](#client-configuration). The only required option to get started is `url`, the base url for your Okta domain.
+## Usage guide
 
-  - `config` - Options that are used to configure the client
+For an overview of the client's features and authentication flows, check out [our developer docs](https://developer.okta.com/docs/guides/okta_auth_sdk). You can also browse the full [API reference documentation](#api-reference).
```

Comment:
I think linking to our usage docs (and not duplicating them here) is the right decision (:+1) but this section was easy to miss. What about expanding it to list a few of the things that the linked usage guide will cover?